.github/workflows/tests.yaml

```
@@ -35,21 +43,25 @@ jobs:
           npm run dev -- ./tests/e2e/schema.json ./tests/e2e/tags/sdk.ts --disable-analytics --mode tags
           npm run dev -- ./tests/e2e/schema.json ./tests/e2e/single-tag-filter/sdk.ts --disable-analytics --mode single --only-tags ItemsHeader
       - name: Start Mockoon CLI
-        uses: mockoon/cli-action@v2
+        uses: mockoon/cli-action@971589e873f22f7c9604e4236d6dfae3164efa6c # v2 - https://github.com/mockoon/cli-action/commit/971589e873f22f7c9604e4236d6dfae3164efa6c
         with:
           # Mockoon local data file or URL
           data-file: './tests/e2e/schema.json'
           port: 3000
-      - uses: grafana/setup-k6-action@v1
-      - uses: grafana/run-k6-action@v1
+      - uses: grafana/setup-k6-action@v1 # zizmor: ignore[unpinned-uses] # Ignore unpinned uses because the action is Grafana's
+      - uses: grafana/run-k6-action@v1 # zizmor: ignore[unpinned-uses] # Ignore unpinned uses because the action is Grafana's
```

```suggestion
      - uses: grafana/setup-k6-action@v1
      - uses: grafana/run-k6-action@v1
```

No need to add that comment. We have a global rule to ignore `grafana/*` repositories, although you may see the error locally because it doesn't read that file.